tests/common/constants.py

```diff
@@ -7,3 +7,8 @@
 IS_BACKEND_TOPOLOGY_KEY = "is_backend_topology"
 # a topology whos name contains the indicator 'backend' will be considered as a backend topology
 BACKEND_TOPOLOGY_IND = "backend"
+# ssh connect default username and password
+DEFAULT_SSH_CONNECT_PARAMS = {
+    "public": {"username": "admin",
+               "password": "YourPaSsWoRd"}
```

Comment:
Can we get the ssh creds from ansible inventory? Then we don't have to hardcoded it in test case

Reply:
We can find some login params in `ansible/group_vars/all/creds.yml`. For SONiC default password, it just lists all passwords but does not has the mapping logic. I think it may try them all when ssh login. But in these two cases especially `test_ssh_default_password`, we want to test SSH connect with the expected password. Thus, we add some adjustment logic to get different default password. To avoid exposure in special test case, we just move the SSH creds into constants, also for reusing. But it can be improved as you say.